tripy/README.md

```diff
@@ -66,39 +66,48 @@ To get the latest changes in the repository, you can build Tripy wheels from sou
 
 <!-- Tripy: DOC: OMIT End -->
 
-## Quickstart
+## Getting Started
 
-In eager mode, Tripy works just like you'd expect:
-
-```py
-# doc: no-print-locals
-import tripy as tp
-
-a = tp.Tensor([1.0, 2.0])
-print(a + 1)
-```
+We've included several guides in Tripy to make it easy to get started.
+We recommend starting with the
+[Introduction To Tripy](https://nvidia.github.io/TensorRT-Incubator/pre0_user_guides/00-introduction-to-tripy.html)
+guide.
 
-Tripy can also compile functions to generate efficient machine code for faster execution:
+To get an idea of the look and feel of Tripy, let's take a look at a short code example.
+All of the features used in this example are explained in more detail in the
+introduction guide mentioned above.
 
 ```py
-# doc: no-print-locals
-def add(a, b):
-    return a + b
-
-# When compiling, we need to specify shape and data type constraints on the inputs:
-
-# a is a 1D dynamic shape tensor of shape (d,), where `d` can range from 1 to 5.
-# `[1, 2, 5]` indicates a range from 1 to 5, with optimization for `d = 2`.
-a_info = tp.InputInfo(shape=([1, 2, 5],), dtype=tp.float32)
-
-# `b` is a 1D tensor of shape (1,).
-b_info = tp.InputInfo((1,), dtype=tp.float32)
-
-compiled_add = tp.compile(add, args=[a_info, b_info])
-
-print(compiled_add(tp.Tensor([1., 2., 3.]), tp.Tensor([3.])))
+# Define our model:
+class Model(tp.Module):
+    def __init__(self):
+        self.conv = tp.Conv(in_channels=1, out_channels=1, kernel_dims=[3, 3])
+
+    def __call__(self, x):
+        x = self.conv(x)
+        x = tp.relu(x)
+        return x
+
+
+# Initialize the model and populate weights:
+model = Model()
+model.load_state_dict(
+    {
+        "conv.weight": tp.ones((1, 1, 3, 3)),
+        "conv.bias": tp.ones((1,)),
+    }
+)
+
+inp = tp.ones((1, 1, 4, 4))
+
+# Eager mode:
+eager_out = model(inp)
+
+# Compiled mode:
+compiled_model = tp.compile(
+    model,
+    args=[tp.InputInfo(shape=(1, 1, 4, 4), dtype=tp.float32)],
+)
+
+compiled_out = compiled_model(inp)
```

Comment:
Should we list few key topics like `dynamic shapes`, `compilation/eager`, `quantization` below and link them to their respective doc? It's always better to highlight the key features when you have users attention.

Reply:
Yes, let me do that in a follow-up PR so that I know what the section URLs are once the new docs are published.